application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildBanDeleteSpamThreadListener.java

```diff
@@ -0,0 +1,43 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.guild.GuildBanEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jooq.Result;
+
+import org.togetherjava.tjbot.commands.EventReceiver;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.HelpThreadsRecord;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import static org.togetherjava.tjbot.db.generated.Tables.HELP_THREADS;
+
+public final class OnGuildBanDeleteSpamThreadListener extends ListenerAdapter
+        implements EventReceiver {
+    private static final Duration recentThreadDuration = Duration.ofMinutes(30);
```

Comment:
30m sounds really short

Reply:
the issue said 10m so i at least did it 30m

Reply:
how much do u think it should be?

Reply:
Nvm, this is fine, I re-read the issue and it was purely talking about spam messages our bot detects.